### ISSUE TITLE
lxRDFa attributes disappear on XM* elements

### ISSUE THREAD

**Issue by dginev** [brucemiller/LaTeXML#866]
I have some `\lxRDFa` macros activating on mutli-line equations, which lead to undesired behavior.

Namely, the `property` and `content` attributes that I use get recorded on the `XMArg` for the current equation branch the exact chain being:

> XMArray -> XMRow -> XMCell -> ... -> XMArg

I tried experimenting with floating the metadata attributes on each of the XM levels above, but it kept getting erased/dropped by the final XML target. Only after I floated it all the way up to the containing `ltx:Math` element did it manage to persist, and appear on the `equationgroup` in the XML and the final `table` element in the follow-up HTML target. 

So, long-story short, I would like to be able to persist metadata on any of the XM* nodes. What would be the best place to enhance that? Alternatively, I am happy to offer a basic quickfix that floats the element up to the main Math element, and agree to not do sub-expression granularity for metadata. That is probably best achieved in the schema, as the `floatToAttribute('property')` call can easily be controlled by forbidding the metadata attributes on XM* elements.

Would appreciate a quick solution here, happy to do the work!

**Comment by dginev** [comment]
I have the feeling I wasn't explicit enough about the exact issue:
 * Even though `\lxRDFa` successfully records metadata attributes on the nearest XM* element, by the tmie the final XML documents is written, those elements are transformed when inside a MathFork context (e.g. equation groups with alignments). So the values recorded on the 4 elements I exemplified in the issue never make it to the written XML document.

**Comment by dginev** [comment]
@brucemiller bump. I would like to get a quick solution in place here, maybe just add on the Authorea fork for now. But would be interested in what your instinct is to the right approach.

**Comment by brucemiller** [comment]
`XMArg` (and usually `XMWrap`) are temporary containers that disappear when their content is successfully parsed. Of course, it's then a bit tricky to merge the attributes from the disappearing `XMArg` onto the replacement node (eg. they both might have id's!).  So, probably the rdf stuff should be listed as something to copy?  Do you have a tiny example to demonstrate?

**Comment by dginev** [comment]
No longer relevant for me, we can push this back until it is needed again.
